src/vs/workbench/contrib/webviewView/browser/webviewViewPane.ts

```diff
@@ -134,14 +134,11 @@ export class WebviewViewPane extends ViewPane {
 
 	protected layoutBody(height: number, width: number): void {
 		super.layoutBody(height, width);
-
-		if (!this._webview.value) {
-			return;
-		}
-
-		if (this._container) {
-			this._webview.value.layoutWebviewOverElement(this._container, new DOM.Dimension(width, height));
-		}
+		setTimeout(() => {
```

Comment:
Does the timeout cause any issues whenclicking the expand and collapse quickly? e.g. < second